src/Logary.Facade/Facade.fs

```diff
@@ -315,6 +457,113 @@ module internal Formatting =
+    go []
+
+  let literateLogLevelTokenizer = function
+    | Debug ->    "DBG", LevelDebug
+    | Error ->    "ERR", LevelError
+    | Fatal ->    "FTL", LevelFatal
+    | Info ->     "INF", LevelInfo
+    | Verbose ->  "VRB", LevelVerbose
+    | Warn ->     "WRN", LevelWarning
+
+  let literateColorizeExceptions (context: LiterateContext) message =
+    let exnExceptionParts =
+      match message.fields.TryFind FieldExnKey with
+      | Some (:? Exception as ex) ->
+        literateExceptionColorizer context ex
+        @ [ Environment.NewLine, Text ]
+      | _ -> [] // there is no spoon
+    let errorsExceptionParts =
+      match message.fields.TryFind "errors" with
+      | Some (:? List<obj> as exnListAsObjList) ->
+        exnListAsObjList |> List.collect (function
+          | :? exn as ex ->
+            literateExceptionColorizer context ex
+            @ [ Environment.NewLine, Text ]
+          | _ -> [])
+      | _ -> []
+
+    exnExceptionParts @ errorsExceptionParts
+
+  /// Split a structured message up into theme-able parts (tokens), allowing the
+  /// final output to display to a user with colours to enhance readability.
+  /// Final output sample:
+  /// "[11:00:01 INF] Hello, world! There are 100 items in your cart: [Item1, Item2, Item3]"
+  /// TODO: sample string*Token list ? too confusing?
+  let literateTokenizer (context: LiterateContext) (message: Message) : (string * LiterateToken) list =
+    let formatLocalTime (utcTicks : int64) =
+      DateTimeOffset(utcTicks, TimeSpan.Zero).LocalDateTime.ToString("HH:mm:ss", context.FormatProvider)
+      , Subtext
+
+    let themedMessageParts =
+      message.value
+      |> literateFormatValue context message.fields
+      |> fun (_, themedParts) -> themedParts
+
+    let themedExceptionParts =
+      if not themedMessageParts.IsEmpty then
+        let exnParts = literateColorizeExceptions context message
+        if not exnParts.IsEmpty then
+          [ Environment.NewLine, Text ]
+          @ exnParts
+          @ [ Environment.NewLine, Text ]
+        else []
+      else []
+
+    [
+      "[", Punctuation
+      formatLocalTime message.utcTicks
+      " ", Subtext
+      literateLogLevelTokenizer message.level
+      "] ", Punctuation
+    ]
+    @ themedMessageParts
+    @ themedExceptionParts
+
+  let literateDefaultColorWriter sem (parts: (string * ConsoleColor) list) =
+    lock sem <| fun _ ->
+      parts |> List.iter (fun (text, color) ->
+        Console.ForegroundColor <- color
+        Console.Write(text)
+      )
+      Console.ResetColor()
+
+/// Logs a line in a format that is great for human consumption,
+/// using console colours to enhance readability.
+/// Sample: [10:30:49 INF] User "AdamC" began the "checkout" process with 100 cart items
+type LiterateConsoleTarget(minLevel, ?context, ?colorize, ?colorWriter, ?consoleSemaphore) =
+  let sem           = defaultArg consoleSemaphore (obj())
```

Comment:
Right, good catch. we can't have two targets writing to the console, messing up the pretty formatting.